src/components/tooltip/tooltip-test.stories.tsx

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable react/prop-types */
+import React, { forwardRef, useState } from "react";
+import Tooltip from ".";
+import { TOOLTIP_POSITIONS, TooltipPositions } from "./tooltip.config";
+import { ButtonProps } from "../button";
+
+export default {
+  title: "Tooltip/Test",
+  includeStories: ["DefaultStory"],
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disableSnapshot: true,
+    },
+  },
+  argTypes: {
+    type: {
+      options: ["error", "default", "info", "warning"],
+      control: {
+        type: "select",
+      },
+    },
+    size: {
+      options: ["medium", "large"],
+      control: {
+        type: "select",
+      },
+    },
+    flipOverrides: {
+      options: ["left", "right", "top", "bottom"],
+      control: {
+        type: "select",
+      },
+    },
+    position: {
+      options: TOOLTIP_POSITIONS,
+      control: {
+        type: "select",
+      },
+    },
+    bgColor: {
+      control: {
+        type: "text",
+      },
+    },
+    fontColor: {
+      control: {
+        type: "text",
+      },
+    },
+  },
+};
+
+interface TooltipStoryProps {
+  message?: React.ReactNode;
+  isVisible?: boolean;
+  flipOverrides?: TooltipPositions;
+}
+
+export const DefaultStory = ({
+  message,
+  isVisible,
+  flipOverrides,
+  ...args
+}: TooltipStoryProps) => {
```

Comment:
Currently the `args` object has an empty type `{}`, because `DefaultStory` is typed to only expect any of the three properties: `message`, `isVisible` and/or `flipOverrides`.

To type `DefaultStory` to expect the `Tooltip` props to be passed too, we need to amend the type definition slightly like this:

```suggestion
}: TooltipProps & TooltipStoryProps) => {
```